iOS Client/Constants/MBConstants.swift

```diff
@@ -14,5 +14,6 @@ struct MBConstants {
     static let DEFAULTS_KEY_BACKGROUND_APP_REFRESH_ATTEMPT_TIMESTAMP = "DEFAULTS_KEY_BACKGROUND_APP_REFRESH_ATTEMPT_TIMESTAMP"
     static let SECONDS_IN_A_WEEK: Double = 7*24*3600
     static let SECONDS_IN_A_DAY: Double = 24*3600
+    static let MAX_ARTICLES_ON_DEVICE: Int = 200
```

Comment:
Is there a reason you chose 200?

Reply:
nope. I was thinking about exposing this to the user through a setting somehow (maybe a low, medium, high setting for how many articles they'd like to persist . . .). There just has to be some cutoff